src/test/java/io/confluent/examples/streams/kafka/KafkaEmbedded.java

```diff
@@ -156,23 +155,22 @@ public void createTopic(final String topic, final int partitions, final int repl
    */
   public void createTopic(final String topic,
                           final int partitions,
-                          final int replication,
-                          final Properties topicConfig) {
+                          final short replication,
+                          final Map<String, String> topicConfig) {
     log.debug("Creating topic { name: {}, partitions: {}, replication: {}, config: {} }",
-        topic, partitions, replication, topicConfig);
-    // Note: You must initialize the ZkClient with ZKStringSerializer.  If you don't, then
-    // createTopic() will only seem to work (it will return without error).  The topic will exist in
-    // only ZooKeeper and will be returned when listing topics, but Kafka itself does not create the
-    // topic.
-    final ZkClient zkClient = new ZkClient(
-        zookeeperConnect(),
-        DEFAULT_ZK_SESSION_TIMEOUT_MS,
-        DEFAULT_ZK_CONNECTION_TIMEOUT_MS,
-        ZKStringSerializer$.MODULE$);
-    final boolean isSecure = false;
-    final ZkUtils zkUtils = new ZkUtils(zkClient, new ZkConnection(zookeeperConnect()), isSecure);
-    AdminUtils.createTopic(zkUtils, topic, partitions, replication, topicConfig, RackAwareMode.Enforced$.MODULE$);
-    zkClient.close();
+      topic, partitions, replication, topicConfig);
+
+    final Properties properties = new Properties();
+    properties.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList());
+
+    try (final AdminClient adminClient = AdminClient.create(properties)) {
+      final NewTopic newTopic = new NewTopic(topic, partitions, replication);
+      newTopic.configs(topicConfig);
```

Comment:
@ableegoldman I think this line answers your question (I _think_ we cannot pass in a `Properties` object here)

Reply:
Yep, that's right.